src/api/lib/feature_switch/feature.rb

```diff
@@ -0,0 +1,13 @@
+module Feature
+  def self.active?(feature)
+    if User.current && (User.current.is_admin? || User.current.is_staff?)
```

Comment:
Zero (more), as we already do this with every request for the Peek status bar in the webui_controller:
https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/webui_controller.rb#L251